routes/infrastructure/config.py

```diff
@@ -0,0 +1,52 @@
+"""Settings for Cloudfront distribution - any environment variables starting with
```

Comment:
Can you add a line to the readme [advanced config](https://github.com/NASA-IMPACT/veda-backend#advanced-configuration) about this? (sorry if this is a repeat comment)

Reply:
Done